### DISCUSSION TITLE
Make tangent to construction geometry

### DISCUSSION THREAD

**Discussion by automaton-v** [DeepSOIC/Lattice2#120] [General]
Is nice feature for attachment, if your object to be arrayed is drawn off centre (for whatever reason) or there are multiple extrusions in one sketch & must find centre between objects. 

You might have to invent construction extrusions

**Comment by DeepSOIC** [comment]
i don't understand you, please explain in more detail

**Comment by DeepSOIC** [comment]
closing for the lack of input
